introduction.md

```diff
@@ -1 +1,4 @@
 Javascript This is my first fork file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?